pkg/fragment/definition.go

```diff
@@ -43,6 +45,29 @@ func Define(path string, options ...DefinitionOption) *Definition {
 	return definition
 }
 
+func (d *Definition) Children() map[string]*Definition {
+	return d.children
+}
+
+func (d *Definition) Child(name string) *Definition {
+	return d.children[name]
+}
+
+func WithChildren(children Children) DefinitionOption {
```

Comment:
Would it be better to have a separate `Define` function that accepts children instead of using the options pattern here?